models/src/local/BasicInformationOverrides.ts

```diff
@@ -10,14 +10,15 @@ LocalMatter.children.push({
     tag: "cluster",
     name: "BasicInformation",
 
+    // We override default in Model to 0 in order to use our default settings.
+    // Both API versions use defaults when 0 would exist.
     children: [
         {
             tag: "attribute",
-            name: "ProductAppearance",
-            id: 0x14,
-            type: "ProductAppearanceStruct",
-            conformance: "O",
-            quality: "F",
+            name: "MaxPathsPerInvoke",
+            id: 0x16,
+            constraint: "min 0",
+            default: 0,
```

Comment:
I think it'd be better to maintain the default in the schema but change it in the cluster when we use it, but not a big deal.

Reply:
How you mean that?

Reply:
Don't create the override but rather modify BasicInformationCluster when we use it (via e.g. `BasicInformationCluster.set`).  Just so the model represents the spec's "truth" but we can be more flexible

Reply:
my problem with this is that I can not determine of the value 1 is from the default or because the dev explicitely wanted to set 1 ... so yes I could so "if ===1 then set to Default aka 10", but i can not determine if the 1 is from dev or from default. Or what do I miss?